packages/cfd/types.ts

```diff
@@ -0,0 +1,22 @@
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+
+export type TNewDetailsOfEachMT5Loginid = Required<
+    Omit<DetailsOfEachMT5Loginid, 'market_type'> & { market_type?: 'financial' | 'synthetic' | 'all' }
+>;
+
+export type TTradingPlatformAvailableAccount = {
```

Comment:
@hamza-deriv You can delete this type as it's the same as `TStores['client']['trading_platform_available_accounts']` 🙇🏻